playbooks/files/create_dcs_instance.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+import openstack
+import sys
+
+conn = openstack.connect()
+
+instance_name = sys.argv[1]
+vpc_id = sys.argv[2]
+subnet_id = sys.argv[3]
+sg_id = sys.argv[4]
+
+instance = conn.dcs.create_instance(
+    name=instance_name,
+    available_zones=["eu-de-03"],
```

Comment:
there's conn.dcs.availability_zones()  which provides  list of available zones for DCS,
please use it to dynamically fetch the AZ instead of hardcoding the AZ.

Reply:
Please see commit 07e9f2391ef056e1236d343fe271b8d783179e47

Reply:
see commit bf77728f5b30df8c3de397d6e05370aabe32ea5e

Reply:
see ca8e7b8d85b1a4da501593294f87ae6ce36c2707